api/AmbientLightSensor.json

```diff
@@ -44,6 +44,17 @@
           "ie": {
             "version_added": false
           },
+          "oculus": {
+            "flags": [
+              {
+                "type": "preference",
+                "name": "#enable-experimental-web-platform-features",
+                "value_to_set": "Enabled"
+              }
+            ],
+            "version_added": "5.0",
+            "notes": "<code>AmbientLightSensor</code> is defined but it never generates any readings."
```

Comment:
I would mark as partially implemented. Same for any other sensors that are defined but don't work.

Alternatively, set version_added false but leave the note. Not sure what's best one of the maintainers will be better placed to make that decision.

I think potentially while it's still flagged just leave it as version added false. And then maybe once its unflagged a decision could be made on how to represent it.

Reply:
I would say as a web developer if it straight up doesn't work and is still flagged the partial with flag is pretty useless information. Once it's unflagged and I can find the interface a note (+ optionally a partial implementation boolean) would be very useful to explain the odd behaviour.